### DISCUSSION TITLE
Can not send message in widget only (error: Your request was not processed as expected )

### DISCUSSION THREAD

**Discussion by Micror** [LiveHelperChat/livehelperchat#2046] [General]
Hi,
Just installed the LHC, new user just got first issue: Can not send message from chat widget of web page, but popup chat window is just fine. Here are system and step:
1) Generated the embed code and pasted in my webpage, 
2) SSL, Https only in my website. 
3) uncheck Use session captcha. LHC have to be installed on the same domain or subdomain.
4) Chat widget showing on webpage, but not able to send message out when using Chrome, Firefox and Windows Edge. However,  Safari is just working fine!
5) When change widget to popup window, the message can be sent without any problem.
6) tracked the error, looks like the error in the following codes lbchatvalidator.php (Captcha validation ):
`        if (!isset($additionalParams['ignore_captcha']) || $additionalParams['ignore_captcha'] == false)
        {
            if (erLhcoreClassModelChatConfig::fetch('session_captcha')->current_value == 1) {
            	if ( !$form->hasValidData( $nameField ) || $form->$nameField == '' || $form->$nameField < time() - 1800 || $hashCaptcha != sha1($_SERVER['REMOTE_ADDR'].$form->$nameField.erConfigClassLhConfig::getInstance()->getSetting( 'site', 'secrethash' ))){
            		$Errors['captcha'] = erTranslationClassLhTranslation::getInstance()->getTranslation("chat/startchat","Your request was not processed as expected - but don't worry it was not your fault. Please re-submit your request. If you experience the same issue you will need to contact us via other means.");
            	}
            } else {
                // Captcha validation
            	if ( !$form->hasValidData( $nameField ) || $form->$nameField == '' || $form->$nameField < time() - 1800 )
            	{
            		**$Errors['captcha'] = erTranslationClassLhTranslation::getInstance()->getTranslation("chat/startchat","Your request was not processed as expected - but don't worry it was not your fault. Please re-submit your request. If you experience the same issue you will need to contact us via other means.");**
            	}
            }
        } `

Please help. Thanks

**Comment by remdex** [comment]
Hi,
It still seems either problem with your env as locally can't be reproduced. You try to find me on discord and write a private message.
